content/en/tracing/trace_collection/compatibility/dotnet-core.md

```diff
@@ -67,13 +67,15 @@ The [latest version of the .NET Tracer][4] can automatically instrument the foll
 | Elasticsearch                   | `Elasticsearch.Net` 5.3.0+                                                                           | `ElasticsearchNet`   |
 | GraphQL .NET                    | `GraphQL` 2.3.0+                                                                                     | `GraphQL`            |
 | gRPC                            | `Grpc.Net.Client`2.30.0+ (.NET Core 3.0+ only)</br>`Grpc.Core` 2.30.0+</br>`Grpc.AspNetCore` 2.30.0+ | `Grpc`               |
+| HotChocolate                    | `HotChocolate.AspNetCore` 11.0.0+                                                                    | `HotChocolate`       |
 | HttpClient / HttpMessageHandler | `System.Net.Http` 4.0+                                                                               | `HttpMessageHandler` |
 | Kafka                           | `Confluent.Kafka` 1.4+                                                                               | `Kafka`              |
 | MongoDB                         | `MongoDB.Driver.Core` 2.1.0+                                                                         | `MongoDb`            |
 | MySql                           | `MySql.Data` 6.7.0+</br>`MySqlConnector` 0.61.0+                                                     | `MySql`              |
 | Oracle                          | `Oracle.ManagedDataAccess` 4.122.0+                                                                  | `Oracle`             |
 | PostgreSQL                      | `Npgsql` 4.0+                                                                                        | `Npgsql`             |
-| RabbitMQ                        | `RabbitMQ.Client` 3.6.9+                                                                             | `RabbitMQ`           |
+| Process                         | `"System.Diagnostics.Process"` 4.0+                                                                  | `Process`            |
+| RabbitMQ                        | `RabbitMQ.Client` 3.6.9+, `EasyNetQ` 1.0.0+                                                          | `RabbitMQ`           |
```

Comment:
Should we leave `EasyNetQ` out? We don't usually include libraries that have a dependency on the instrumented library.

For example, we list `Elasticsearch.Net` (the lower-level API we instrument), but not `NEST` (the higher-level strongly-typed library). We also list ADO.NET but not Entity Framework.

Reply:
Maybe the same comment for `HotChocolate.AspNetCore` vs `HotChocolate`?

Reply:
I don't know. I wanted to somehow show that now that we instrument method implementing interfaces, we can also have more coverage. Maybe we should ask the support team as they are the ones who see those questions the most i suppose. @link04 for instance, any input here?

Reply:
Yeah I think for the ones we don't instrument directly are better to leave out, otherwise user's may reach out expecting full support and are confused when they don't get spans for certain methods(also happens with Java for [example](https://docs.datadoghq.com/tracing/trace_collection/compatibility/java/#web-framework-compatibility)). 
![image](https://user-images.githubusercontent.com/28125428/201971893-ce06dcb9-7086-4967-a195-e05ea4ebb26c.png)

I think we could either add a section saying they could see spans for *x* or *y* scenarios on this not directly instrumented libraries, we could do so there in the public facing docs or internals so SE's can find details when they look at the KB(safer side for us in the future), wdyt?

Reply:
So I'm removing this and I'll make sure we add something in the internal doc. If it makes sense to have it public, we can revisit